### PR TITLE
integration of calender app

### PR DESCRIPTION
**Pull request by MuskanSharma20**
have integrated the calender app.

### CODE REVIEW

File: client/dashboard/src/component/Calender.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
+import FullCalendar from "@fullcalendar/react";
+import daygridPlugin from "@fullcalendar/daygrid";
+import interactionPlugin from "@fullcalendar/interaction";
+import { useState, useEffect } from "react";
+import { Modal, Button, Form } from "react-bootstrap";
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (reerajput930) [inline client/dashboard/src/component/Calender.js]: comment out -> import 'bootstrap/dist/css/bootstrap.min.css';

Comment (reerajput930) [reply client/dashboard/src/component/Calender.js]: then push the code again and create pull request.

**Comment by MuskanSharma20** [comment]
commented bootsrap css

**Comment by reerajput930** [comment]
Make a branch of the repo and then send me the pull request. Name the branch as ` muskan `

**Pull request closed without merging**